### PR TITLE
Update styles in leftbar and change link in settings

### PR DESCRIPTION
**Pull request by jclopezdev**
-  Add styles in leftbar items
- Remove br's

### CODE REVIEW

File: app/views/partials/_leftbar.html.haml [hunk 1]
```diff
@@ -5,32 +5,24 @@
...
-  %br/
+%ul#slide-out.side-nav.fixed.grey.darken-4.blue-grey-text.text-lighten-4
   #profilepic
-    = image_tag current_user.photo.url(:thumb) || current_user.image, size: "110x110", class: 'circle responsive-img', onerror: "this.onerror=null;this.src='" + current_user.gravatar_url + "?type=large';"
-    %br/
-    %br/
-    = current_user.name ||= current_user.email
-  %br/
-  %br/
+    %img.circle.responsive-img{:alt => "", :src => (current_user.image ||= current_user.gravatar_url) + "?type=large", :width => "150", :height => "150" }/
```
Comment (edman193) [inline app/views/partials/_leftbar.html.haml]: you can use "src: ()" instead of :src => ()

File: app/views/partials/_leftbar.html.haml [hunk 2]
```diff
@@ -5,32 +5,24 @@
...
+    .profile-name
+      = current_user.name ||= current_user.email
   %li
-    %a.blue-text.text-lighten-2{href: user_root_path }
-      %i.fa.fa-home.fa-2x.fa-fw>
-      Home
+    = link_to '<i class="fa fa-home fa-2x fa-fw"></i> <span>Home</span>'.html_safe,
+      user_root_path, :class => 'grey-text darken-3'
```
Comment (edman193) [inline app/views/partials/_leftbar.html.haml]: you can use class: "" instead of :class => ""

File: app/views/partials/_leftbar.html.haml [hunk 3]
```diff
@@ -5,32 +5,24 @@
...
   %li
-    %a#links.pink-text.lighten-2{href: beers_path }
-      %i.fa.fa-beer.fa-2x.fa-fw>
-      Beers
+    = link_to '<i class="fa fa-beer fa-2x fa-fw"></i> <span>Beers</span>'.html_safe,
+      beers_path, :class => 'grey-text darken-3'
```
Comment (edman193) [inline app/views/partials/_leftbar.html.haml]: you can use class: "" instead of :class => ""

File: app/views/partials/_leftbar.html.haml [hunk 4]
```diff
@@ -5,32 +5,24 @@
...
   %li
-    %a#links.green-text.text-lighten-2{href: url_for(controller: "users", id: current_user, action: "edit")}
-      %i.fa.fa-cog.fa-2x.fa-fw>
-      Settings
+    = link_to '<i class="fa fa-cog fa-2x fa-fw"></i> <span>Settings</span>'.html_safe,
+      edit_user_path(current_user), :class => 'grey-text darken-3'
```
Comment (edman193) [inline app/views/partials/_leftbar.html.haml]: you can use class: "" instead of :class => ""

File: app/views/partials/_leftbar.html.haml [hunk 5]
```diff
@@ -5,32 +5,24 @@
...
   - if current_user.has_role?(:admin)
     %li
-      %a#links.red-text.text-lighten-1{href: admin_users_path}
-        %i.fa.fa-lock.fa-2x.fa-fw>
-        Admin
+      = link_to '<i class="fa fa-lock fa-2x fa-fw"></i> <span>Admin</span>'.html_safe,
+      admin_users_path, :class => 'grey-text darken-3'
```
Comment (edman193) [inline app/views/partials/_leftbar.html.haml]: you can use class: "" instead of :class => ""

File: app/views/partials/_leftbar.html.haml [hunk 6]
```diff
@@ -5,32 +5,24 @@
...
   %li
-    = link_to '<i class="fa fa-code fa-2x fa-fw"></i> Log out'.html_safe, destroy_user_session_path, method: :delete, class: 'links.pink-text.text-lighten-2'
+    = link_to '<i class="fa fa-sign-out fa-2x fa-fw"></i> <span>Log out</span>'.html_safe,
+      destroy_user_session_path, method: :delete, :class => 'grey-text darken-3'
```
Comment (edman193) [inline app/views/partials/_leftbar.html.haml]: you can use class: "" instead of :class => ""

**Comment by jclopezdev** [comment]
Ready! @edman193

**Pull request merged**